vue-frontend/src/components/home/NewCTASection.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div class="tw-relative tw-top-10 sm:tw-top-px tw-pt-12 tw-font-inter-medium">
+    <div class="tw-flex tw-flex-col tw-items-center">
+      <img
+        :src="Cta2400"
+        :srcset="`${Cta400} 400w, ${Cta800} 800w, ${Cta1200} 1200w, ${Cta2400} 2400w`"
```

Comment:
background image is blur and its circles shapes are getting change in most of devices which is not looking good